guides/release/models/finding-records.md

```diff
@@ -2,34 +2,36 @@ The EmberData store provides an interface for retrieving records of a single typ
 
 ### Retrieving a Single Record
 
-Use [`store.findRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/findRecord?anchor=findRecord) to retrieve a record by its type and ID.
-This will return a promise that fulfills with the requested record:
+Use [`findRecord()`](https://api.emberjs.com/ember-data/5.3/functions/@ember-data%2Fjson-api%2Frequest/findRecord) to retrieve a record by its type and ID.
+This will return a single `record` or a `array` depends upon the response from server:
```

Comment:
we should probably introduce builders / the concept of the request first, because this function won't do this on its own and this particular one builds a json:api specific request.

Reply:
Umm, that Makes sense. But the builder or the concept of the request will be in a separate document, right? Since this is specific to finding records.

Reply:
there's a chicken/egg problem, unsure if its better in a separate file or not because its likely you want to introduce them together